interview-questions/activity-interview-questions.md

```diff
@@ -0,0 +1,70 @@
+# Activity: Interview Questions
+
+## Goal
+
+Our goal is to compile a series of interview question resources and interview questions. By the end of this activity, everyone should have access to some recommended interview questions.
+
+## Preparation
+
+The facilitator should make a spreadsheet that includes everyone's name, and allows students to notate at least two questions. For example:
+
+| Name | Question #1 | Question #2 |
+| ---- | ----------- | ----------- |
+|      |             |
+
+The facilitator should also prepare a way to record ideas in part one. Some recommended methods include:
+
+- The word cloud feature of [Mentimeter](https://www.mentimeter.com/)
+- Taking notes on a shared document
+
+## Activity Instructions
+
+There are two parts to this activity:
+
+1. Crowdsource interview resources
+1. Crowdsource interview questions
+
+### Crowdsource Resources
+
+As a large group, follow directions from the facilitator to create a shared list of interview question resources.
+
+These resources may include:
+
+- Websites made for technical interview preparation
+- Newsletters made for technical interview preparation
+- Recommended books
+
+If it includes practice interview questions, then it's valid!
+
+### Crowdsource Questions
+
+Afterwards, individually find and share practice questions. Everyone must contribute at least _two_ questions to the shared document of practice questions.
+
+- One must be a coding question
+- The second may be a behavioral question
+- There can be overlap or repetition compared to other entries in the document.
+
+For each coding question, include a link to its original source if possible. For example:
+
+| Name         | Question #1                                                                 | Question #2                                                                                                         |
+| ------------ | --------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| Eamonn Finch | [Roman Numeral to Integer](https://leetcode.com/problems/roman-to-integer/) | Describe a time when you were working with someone and there was a conflict. What happened, and how did it resolve? |
```

Comment:
I like this phrasing a little better. It doesn't assume the respondent was the one who resolved the situation, but still sounds less like the situation just resolved itself.
```suggestion
| Eamonn Finch | [Roman Numeral to Integer](https://leetcode.com/problems/roman-to-integer/) | Describe a time when you were working with someone and there was a conflict. What happened, and what was the resolution? |
```